src/WebJobs.Extensions.MobileApps/Bindings/MobileTableQueryValueProvider.cs

```diff
@@ -2,26 +2,25 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System;
-using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
 using Microsoft.Azure.WebJobs.Host.Bindings;
 using Microsoft.WindowsAzure.MobileServices;
 
 namespace Microsoft.Azure.WebJobs.Extensions.MobileApps
 {
-    internal class MobileTableQueryValueProvider<T> : IValueProvider
+    internal class MobileTableQueryValueProvider<T> : IValueBinder
```

Comment:
Why do you still have ValueBinder if SetValue() is a nop? Can we get rid of this class?